README.md

```diff
@@ -20,7 +20,7 @@ This capstone consists of the mobile and desktop versions with the following sec
 - JAVASCRIPT
```

Comment:
- [ ] _Here on the README file but not this line._
1. Please fix the screenshot of the app as the previous reviewer asked 😄 🙏 This is for a completely professional and good-looking repo 😉

Reply:
Hello @MahmoudBakr23,
My first submission had the right image link for the screenshot, I am sure the first reviewer checked the main instead of the branch I worked on that was why the screenshot link seems broken. However, she recommended I fix or remove the screenshot link so I removed it but now I have added it back. Below is an image that shows the link is not broken. Kindly go through it. Thank you.

![Screenshot from 2021-09-11 07-37-43](https://user-images.githubusercontent.com/58692590/132939127-cc0f72e9-43df-4d16-998f-54b849fa5480.png)